### PR TITLE
Feature/subscribers

### PR DESCRIPTION
**Pull request by jerry1ye10**
added subscribers scaffold

### CODE REVIEW

**Comment by jkao1** [comment]
I am not familiar with mailing. @NicholasLYang will have to review this.

File: .idea/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NicholasLYang) [inline .idea/.generators]: Don't check out .idea

Comment (NicholasLYang) [reply .idea/.generators]: Remove it from the repo

File: app/mailers/email_subscriber_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class EmailSubscriberMailer < ApplicationMailer
+  default :from => 'any_from_address@example.com'
```
Comment (NicholasLYang) [inline app/mailers/email_subscriber_mailer.rb]: Set default address to info@stuyspec.com

File: app/views/layouts/mailer.html.erb [hunk 3]
```diff
@@ -9,5 +9,6 @@
 
   <body>
     <%= yield %>
+  ewqewqeq
```
Comment (NicholasLYang) [inline app/views/layouts/mailer.html.erb]: What

**Comment by jkao1** [comment]
@NicholasLYang  could you fix the conflicts? Not familiar with this code.

**Pull request closed without merging**